app/src/main/res/values/strings.xml

```diff
@@ -889,6 +891,8 @@
     </string>
     <!-- Text for the button to read the privacy notice -->
     <string name="onboarding_privacy_notice_read_button">Read our privacy notice</string>
+    <!-- Text for closing the onboarding screen -->
```

Comment:
Nit: All comments for content descriptions follow this convention:
<!-- Content description (not visible, for screen readers etc.): Close onboarding screen -->

Reply:
Updated!  Thank you!